source/guide_strapi.rst

```diff
@@ -0,0 +1,299 @@
+.. author:: Richard Henkenjohann <richardhenkenjohann@googlemail.com>
+
+.. tag:: lang-nodejs
+.. tag:: web
+.. tag:: backend
+.. tag:: api
+
+.. highlight:: console
+
+.. sidebar:: Logo
+
+  .. image:: _static/images/strapi.svg
+      :align: center
+
+#####
+Strapi
+#####
+
+.. tag_list::
+
+Strapi_ is a open source headless CMS to build APIs. It’s 100% Javascript, fully customizable and developer-first.
+
+----
+
+.. note:: For this guide you should be familiar with the basic concepts of
+
+  * :manual:`Node.js <lang-nodejs>` and its package manager :manual_anchor:`npm <lang-nodejs.html#npm>`
+  * :manual:`MySQL <database-mysql>`
+  * :manual:`supervisord <daemons-supervisord>`
+  * :manual:`domains <web-domains>`
+
+Prerequisites
+=============
+
+We're using :manual:`Node.js <lang-nodejs>` in the stable version 10:
+
+::
+
+ [isabell@stardust ~]$ uberspace tools version use node 10
+ Using 'Node.js' version: '10'
+ Selected node version 10
+ The new configuration is adapted immediately. Patch updates will be applied automatically.
+ [isabell@stardust ~]$
+
+.. include:: includes/my-print-defaults.rst
+
+Your URL needs to be setup:
+
+.. include:: includes/web-domain-list.rst
+
+Installation
+============
+
+Start with a local project
+--------------------------
+
+When Strapi is in production mode (which should be targeted on Uberspace), the Content Type Creator is disabled. This requires us to build the project locally to be able to make changes on the API and deploy it for production on Uberspace.
+
+We will start with creating a new Strapi project. Use yarn to install the Strapi project (locally).
+
+.. code-block:: console
+
+ [local@My-Computer ~]$ yarn create strapi-app my-project --quickstart
+
+
+.. note::
+
+    When you use --quickstart to create a Strapi project locally, a SQLite database is used. We will later configure a MySQL database that is used in production mode on Uberspace.
+
+
+Init a Git repository
+---------------------
+
+We will initialize a Git repository and commit our initial project files.
+
+.. code-block:: console
+
+ cd my-project
+ git init
+ git add .
+ git commit -m "Initial Commit"
+
+Push your changes to a remote repository, like GitHub. We will later clone this repository.
+
+.. code-block:: console
+
+ git branch -M main
+ git remote add origin git@github.com:me/my-project.git
```

Comment:
Please just use the uberspace itself as a repo here. Feel free to add a `.. note::` box telling users that they can whatever git service they like. This also enables you to skip the server-side SSH key setup.

```suggestion
 git remote add origin strapi@elara.uberspace.de:/home/strapi/my-project.git
```

Create the repo beforehand by running the following
```
git init --bare ~/my-project.git
git symbolic-ref HEAD refs/heads/main
```
on the server. You can later clone it like normal:
```
git clone ~/my-project.git
```